scripts/audit.js

```diff
@@ -0,0 +1,90 @@
+// Check possible npm dependency vulnerabilities
+// Usage: execute "yarn run audit" in the shell
+//
+// You can add exceptions through .nsprc file, which is a hidden file in root folder.
+//
+// Note: to use this script, you should pipe in the output of 'yarn audit --json'
+
+const bfj = require('bfj');
+const fs = require('fs');
+
+const INDENT = '  ';
+const isAuditAdvisory = o => o.type === 'auditAdvisory';
+
+// get an advisory or empty object
+const getAdvisory = o => (!o ? {} : !o.data ? {} : !o.data.advisory ? {} : o.data.advisory);
+// get a resolution or empty object
+const getResolution = o => (!o ? {} : !o.data ? {} : !o.data.resolution ? {} : o.data.resolution);
+
+// Read exceptions
+let exceptions = [];
+bfj
+  .parse(fs.createReadStream('.nsprc'), {})
```

Comment:
You can also just import a JSON file.

Reply:
I didn't want to save any file anywhere. It's a huge file - that's pretty much unreadable for humans due to the size of printed JSON.

Reply:
This was different import :)

I just decided that it's better to import both the same way. Plus I thought that it might be a good discussion point if the exception list should actually be `.auditrc` file with JavaScript syntax since that would allow comments (unlike JSON).

Reply:
Are you talking about a different file?

Reply:
At least the `.nsprc` name is misleading as NSP isn't visible being used at all. Comments in JSON work fine in Node, but it's a subjective decision if you don't want to do that.

Anyways, I trust your decision here, no need to go back and forth any further.

Reply:
Didn't know about node allowing comments :o. I was actually annoyed by that. (However, I'm not sure if bfj allows comments.